.buildkite/ftr_platform_stateful_configs.yml

```diff
@@ -2,11 +2,11 @@ disabled:
   # Stateful base config for deployment-agnostic tests
```

Comment:
@gsoldevila I remember you mentioned that the automation you use would automatically handle updates in the CodeQL "ignore" config, but I don’t see any changes in the CodeQL config in this PR.... Wouldn't this change break [this](https://github.com/elastic/kibana/blob/main/.github/codeql/codeql-config.yml#L87) (i.e. relocated tests will no longer be excluded by CodeQL)?

Reply:
I had to disable reference replacement for this particular relocation.
:onedoesnotsimply: replace all occurrences of `test` string.

Thanks for the heads up, I'll update this one manually